src/meta/greedy_load_balancer.cpp

```diff
@@ -827,6 +839,20 @@ void greedy_load_balancer::greedy_balancer(const bool balance_checker)
         }
     }
 
+    if (!_balance_cluster) {
+        app_balancer(balance_checker);
+        return;
+    }
+
+    if (!balance_checker) {
```

Comment:
what is balance_checker?

Reply:
It is used to get the balancer operation count